src/test/java/org/reactome/addlinks/test/TestEnsemblBioMartFileProcessor.java

```diff
@@ -0,0 +1,69 @@
+package org.reactome.addlinks.test;
+
+import org.junit.Test;
+import org.reactome.addlinks.fileprocessors.ensembl.EnsemblBioMartFileProcessor;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class TestEnsemblBioMartFileProcessor {
+
+    private static final String uniprotToProteinsKey = "hsapiens_uniprotToProteins";
+    private static final String transcriptToMicroarraysKey = "hsapiens_transcriptToMicroarrays";
+    private static final String proteinToGenesKey = "hsapiens_proteinToGenes";
+    private static final String proteinToTranscriptsKey = "hsapiens_proteinToTranscripts";
+    private static final int EXPECTED_RESULTS_LENGTH = 50;
+    private static final String testDirectory = "/tmp/test_ensembl_biomart_mapping_service/";
+    private static final Path improperlyFormattedFilepath = Paths.get(testDirectory + "hsapiens_improperly_formatted_microarray");
+    private static final List<String> improperlyFormattedLines = new ArrayList<>(Arrays.asList("ENSG00000174953ENST00000308361ENSP00000309296ILMN_1738272\n", "ENSG00000230791ENST00000414519ENSP00000400138ILMN_1717127\n"));
```

Comment:
Not to be too nit-picky, but class level constants usually use CAPITALIZED_UNDERSCORE_SEPARATED names :).

Reply:
No, that's not too much of a nit-pick. It's a near-universal Java standard practice.

Reply:
Done